packages/typescript-to-proptypes/test/reconcile-prop-types/input.d.ts

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 
 interface Props {
-  children: React.ReactNode;
+  children?: React.ReactNode;
```

Comment:
Why did you change the test?

Reply:
This wasn't intended. I will revert. Thanks for raising :)